.github/workflows/release.yml

```diff
@@ -135,7 +135,7 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
         tag_name: ${{ needs.create-release.outputs.nixpacks_version }}
-        files: nixpacks-${{ needs.create-release.outputs.nixpacks_version }}-${{ matrix.target }}.tar.gz
+        files: nixpacks-${{ needs.create-release.outputs.nixpacks_version }}-${{ matrix.target }}.{zip,tar.gz}
```

Comment:
```suggestion
        files: nixpacks-${{ needs.create-release.outputs.nixpacks_version }}-${{ matrix.target }}*
```

Reply:
I think its better to use a wildcard to fix https://github.com/railwayapp/nixpacks/pull/134#issuecomment-1127878630